SQL/migrate/V010__Preferences_secondary_toggles.sql

```diff
@@ -0,0 +1,7 @@
+--
+-- Implemented in PR #3653.
+-- Renames parallax toggles to a more generic name.
+--
+
+ALTER TABLE `ss13_characters`
+  CHANGE `parallax_togs` `toggles_secondary` INT(11) NULL DEFAULT NULL;
```

Comment:
woups this is my bad it should be ``parallax_toggles`` instead of ``parallax_togs``

Reply:
Whoopsies

Reply:
Fixed.